### PR TITLE
Migration of the geonames import

### PR DESCRIPTION
**Pull request by flotho**

### CODE REVIEW

File: base_location_geonames_import/README.rst [hunk 1]
```diff
@@ -38,6 +38,7 @@ Contributors
 * Alexis de Lattre <alexis.delattre@akretion.com>
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Florent Thomas <florent.thomas@mind-and-go.com>
```
Comment (pedrobaeza) [inline base_location_geonames_import/README.rst:41]: Well, what you have made is not too much for being added here, don't you think?

**Comment by pedrobaeza** [comment]
A test would help to keep the coverage and be a good contribution

**Comment by simahawk** [comment]
BTW: it's working fine, I tested it w/ l10n_it_base_location_geonames_import (to which I applied almost the same changes).

**Comment by pedrobaeza** [comment]
Superseeded by https://github.com/OCA/partner-contact/pull/253

**Pull request closed without merging**